### ISSUE TITLE
lxcfs not starting since 4.0.4

### ISSUE THREAD

**Issue by lemonzest79** [ganto/copr-lxc4#3]
Hiya, running fedora 32 with your copr enabled and getting this from systemctl status lxcfs.service

lxcfs.service - FUSE filesystem for LXC
     Loaded: loaded (/usr/lib/systemd/system/lxcfs.service; enabled; vendor preset: disabled)
     Active: failed (Result: exit-code) since Sat 2020-07-04 07:27:23 BST; 2h 13min ago
       Docs: man:lxcfs(1)
    Process: 966 ExecStart=/usr/bin/lxcfs /var/lib/lxcfs (code=exited, status=1/FAILURE)
    Process: 968 ExecStopPost=/bin/fusermount -u /var/lib/lxcfs (code=exited, status=203/EXEC)
   Main PID: 966 (code=exited, status=1/FAILURE)

Jul 04 07:27:23 electronicbox systemd[1]: lxcfs.service: Scheduled restart job, restart counter is at 5.
Jul 04 07:27:23 electronicbox systemd[1]: Stopped FUSE filesystem for LXC.
Jul 04 07:27:23 electronicbox systemd[1]: lxcfs.service: Start request repeated too quickly.
Jul 04 07:27:23 electronicbox systemd[1]: lxcfs.service: Failed with result 'exit-code'.
Jul 04 07:27:23 electronicbox systemd[1]: Failed to start FUSE filesystem for LXC.

**Comment by lemonzest79** [comment]
rolled back to fedora 32 provided 4.0.1 and it starts up fine, as did previous lxc4 copr build

**Comment by ganto** [comment]
I can reproduce the error also on Fedora 31. lxcfs fails to start with:
```
Jul 05 23:08:14 formosa.oasis.home systemd[1]: Started FUSE filesystem for LXC.
Jul 05 23:08:14 formosa.oasis.home lxcfs[184173]: /usr/lib64/lxcfs/liblxcfs.so: undefined symbol: lxcfs_clone - Failed to open liblxcfs.so
Jul 05 23:08:14 formosa.oasis.home systemd[1]: lxcfs.service: Main process exited, code=exited, status=1/FAILURE
```
Not exactly sure what's going on here. Will investigate...

**Comment by ganto** [comment]
Created upstream issue lxc/lxcfs#420

**Comment by brauner** [comment]
Sent a fix for this just now.

**Comment by ganto** [comment]
I just released `lxcfs-4.0.4-0.2` with the upstream fix included. Now I cannot reproduce the issue any more.

**Comment by brauner** [comment]
> I just released `lxcfs-4.0.4-0.2` with the upstream fix included. Now I cannot reproduce the issue any more.

Excellent, thank you!

**Comment by lemonzest79** [comment]
Loads up fine now :)

lxcfs.service - FUSE filesystem for LXC
     Loaded: loaded (/usr/lib/systemd/system/lxcfs.service; enabled; vendor preset: disabled)
     Active: active (running) since Tue 2020-07-07 06:15:26 BST; 8h ago
       Docs: man:lxcfs(1)
   Main PID: 832 (lxcfs)
      Tasks: 3 (limit: 57704)
     Memory: 1.7M
     CGroup: /system.slice/lxcfs.service
             └─832 /usr/bin/lxcfs /var/lib/lxcfs

Jul 07 06:15:26 electronicbox lxcfs[832]: - proc_diskstats
Jul 07 06:15:26 electronicbox lxcfs[832]: - proc_loadavg
Jul 07 06:15:26 electronicbox lxcfs[832]: - proc_meminfo
Jul 07 06:15:26 electronicbox lxcfs[832]: - proc_stat
Jul 07 06:15:26 electronicbox lxcfs[832]: - proc_swaps
Jul 07 06:15:26 electronicbox lxcfs[832]: - proc_uptime
Jul 07 06:15:26 electronicbox lxcfs[832]: - shared_pidns
Jul 07 06:15:26 electronicbox lxcfs[832]: - cpuview_daemon
Jul 07 06:15:26 electronicbox lxcfs[832]: - loadavg_daemon
Jul 07 06:15:26 electronicbox lxcfs[832]: - pidfds
